genie-client/src/main/java/com/netflix/genie/client/JobClient.java

```diff
@@ -482,7 +482,33 @@ public InputStream getJobStderr(
         }
         final ResponseBody body = this.jobService.getJobStderr(jobId).execute().body();
         if (body == null) {
-            throw new GenieClientException("No data for job std out returned");
+            throw new GenieClientException("No data for job std err returned");
+        }
+        return body.byteStream();
+    }
+
+    /**
+     * Method to fetch an output file for a job from Genie.
+     *
+     * @param jobId The id of the job whose output file is desired.
+     * @param outputFilePath The path to the file within output directory.
+     * @return An input stream to the output file contents.
+     * @throws GenieClientException If the response received is not 2xx.
+     * @throws IOException          For Network and other IO issues.
+     */
+    public InputStream getJobOutputFile(
```

Comment:
Should `getJobStdErr` and `getJobStdOut` be re-implemented using this, rather than duplicate?